RedHatInsights/compliance-frontend
# feat(Tailorings): RHINENG-14333 - Implement "Selected filter" for rules tables

This PR implements the "Selected Only" toggle for the `Tailorings` component. It can be enabled by passing the `enableSecurityGuideRulesToggle` prop.

Additionally a `profiles` array can now be provided to the Tailorings component as a prop, consisting of objects containing osMajorVersion, osMinorVersion, securityGuideId, profileId and optionally ruleIds, to show additional tailorings to configure. 

**How to test**

1) Open the create policy wizard
2) complete the Wizard till the "Rules" step
3) Verify the "Selected Only" toggle is visible
4) Verify that only "profile rules" are requested
5) Toggle of the "Selected Only" and verify that only "security guide rules" are requested. (Compare the total in the tree view)
6) Verify that the selection is still working as expected


## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
